index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+<body>
+    
+    <!-- navbar section -->
+<nav class="navbar">
+    <h3 id="logo">Dela McRaph</h3>
+    <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+    </ul>
+
+</nav>
+
+<!-- self introduction -->
+<div class="container">
+     <div class="intro"> 
+        <h4 class="primary-text">
+            I AM RAPHAEL. <br>
+            GLAD TO SEE YOU!
+        </h4>

```

Comment:
- [ ] It would really great if you use `h1` over `h4` because it is the main heading of your portfolio and also consider as best practice by implementing the correct semantics. Kindly take a look at the below example.

 ``` suggestion
        <h1 class="primary-text">I AM RAPHAEL. <br />
        GLAD TO SEE <br /> 
        YOU!</h4>

```

### Design should match with this

![Screenshot from 2023-04-27 19-46-44](https://user-images.githubusercontent.com/66476329/234890396-48152d6e-ac72-48fe-a338-51ffd03bd3b9.png)